demo

```diff
@@ -172,7 +194,7 @@ docker-compose \
     -f docker-compose.storage.minio.yml \
     -f docker-compose.demo.yml \
     -p ${DOCKER_COMPOSE_PROJECT_NAME} \
-    $CLIENT \
+    $EXTRA_FILES \
```

Comment:
Aren't we missing here `USER_EXTRA_FILES` ? Not that I have a good use case for it (it was used basically for enterprise). But if we keep supporting it in the script I think we should then add it to all `docker-compose` calls.

Reply:
I guess the docker-compose calls should be collected into one place somewhow then?